bin/reth/src/config.rs

```diff
@@ -0,0 +1,16 @@
+//! CLI command to write default config to stdout
+use clap::Parser;
+use reth_staged_sync::Config;
+
+/// `reth config` command
+#[derive(Debug, Parser)]
+pub struct Command;
```

Comment:
we need a path argument, same as:

https://github.com/paradigmxyz/reth/blob/087d0a63170b944dc70435c691ec54069ccfeea2/bin/reth/src/node/mod.rs#L81-L83